### ISSUE TITLE
invalid argument error when you pass full path in --values

### ISSUE THREAD

**Issue by kuisathaverat** [updatecli/updatecli#1253]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The pipeline fails when you pass the full path for the values file

```
>  updatecli manifest show --config .ci/update-versions/update-helm-charts.yaml --values /Users/myuser/src/a-long-repository-name/.ci/update-versions/helm-charts.yaml


+++++++++++
+ PREPARE +
+++++++++++

Loading Pipeline ".ci/update-versions/update-helm-charts.yaml"
ERROR: failed loading pipeline(s)
        * ".ci/update-versions/update-helm-charts.yaml" - stat /Users/myuser/src/a-long-repository-name/.ci/update-versions/helm-charts.yaml: invalid argument
```

### Expected Behavior

The same pipeline pass in `0.44.4`, and pass in `0.45.0` if you pass a relative path

```
>  updatecli manifest show --config .ci/update-versions/update-helm-charts.yaml --values .ci/update-versions/helm-charts.yaml


+++++++++++
+ PREPARE +
+++++++++++

Loading Pipeline ".ci/update-versions/update-helm-charts.yaml"

SCM repository retrieved: 1
```

### Steps To Reproduce

1 - create a `/tmp/values.yaml` file (replace the values for valid values to checkout a repo)
```yaml
Email: my-email@gmail.com
Owner: owner
Repository: repo
Token: gho_manythings
User: full name
```
2 - create a `/tmp/manifest.yaml` file
```yaml
---
title: Update
scms:
  githubConfig:
    kind: "github"
    spec:
      user: '{{ .User }}'
      email: '{{ .Email }}'
      owner: "{{ .Owner }}"
      repository: "{{ .Repository }}"
      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
      username: '{{ .User }}'
      branch: "main"
```
3 - go to the `/tmp` folder
4 execute `updatecli manifest show --config manifest.yaml --values /tmp/values.yaml` it will fail with an invalid argument error
```
> updatecli manifest show --config manifest.yaml --values /tmp/values.yaml


+++++++++++
+ PREPARE +
+++++++++++

Loading Pipeline "manifest.yaml"
ERROR: failed loading pipeline(s)
        * "manifest.yaml" - stat /tmp/values.yaml: invalid argument

0 pipeline(s) successfully loaded

SCM repository retrieved: 0


++++++++++++++++++
+ AUTO DISCOVERY +
++++++++++++++++++

ERROR: ✗ no valid pipeline found
ERROR: command failed
```
5 execute `updatecli manifest show --config manifest.yaml --values ./values.yaml` it will run
```
> updatecli manifest show --config manifest.yaml --values ./values.yaml


+++++++++++
+ PREPARE +
+++++++++++

Loading Pipeline "manifest.yaml"

SCM repository retrieved: 1
...
```

### Environment

```markdown
- OS: Ubuntu 20.04, macOS 13.3
- updatecli: v0.45.0, v0.48.0
```

### Anything else?

I think is related to https://github.com/updatecli/updatecli/issues/1194

**Comment by olblak** [comment]
Thanks for reporting, I'll try to find some time to look into this.
The root cause is probably due to https://github.com/updatecli/updatecli/pull/1177 
As that part was refactored to allow mock testing

**Comment by olblak** [comment]
I did some testing, and this pull request https://github.com/updatecli/updatecli/pull/1447 solves the current issue.

It's gonna be available in the next release 0.56.0
